NorthstarDedicatedTest/bansystem.cpp

```diff
@@ -20,7 +21,25 @@ void ServerBanSystem::OpenBanlist()
 	{
 		std::string line;
 		while (std::getline(enabledModsStream, line))
-			m_vBannedUids.push_back(strtoull(line.c_str(), nullptr, 10));
+		{
+			// ignore line if first char is #
+			if (line.front() == BANLIST_COMMENT_CHAR)
+				continue;
+
+			// remove tabs which shouldnt be there but maybe someone did the funny
+			line.erase(std::remove(line.begin(), line.end(), '\t'), line.end());
+			// remove spaces to allow for spaces before uids
+			line.erase(std::remove(line.begin(), line.end(), ' '), line.end());
+
+			// check if line is empty to allow for newlines in the file
+			if (line == "")
```

Comment:
Remove this as it's checked for with the other check up top

Reply:
the problem i have with your 2 changes is that this would allow lines that arent `""` but instead `" "` or `"\t"` or any combination of whitespaces really to pass through and cause errors

Reply:
Ah touche, I think in my head I was thinking of it as a `is_whitepace` check